docs/iot/guides/things-api-guide-5.0.md

```diff
@@ -0,0 +1,443 @@
+# API Usage
+
+SmartThings Thing API is a service that provides connection to SmartThings Cloud. Also, it acts as an interface for resource applications to be controlled by the SmartThings applications.
```

Comment:
Please rephrase it as:

In addition, it acts as an interface for resource applications to be controlled by the SmartThings applications.

Reply:
Ok. Done.